packages/in_app_purchase/in_app_purchase_android/example/README.md

```diff
@@ -7,3 +7,57 @@ package.
 
 Unless you are making changes to this implementation package, this example is
 very unlikely to be relevant.
+
+# In App Purchase Example
+
+### Preparation
+
+There's a significant amount of setup required for testing in-app purchases
+successfully, including registering new app IDs and store entries to use for
+testing in the Play Developer Console. Google Play requires developers to
+configure an app with in-app items for purchase to call their in-app-purchase
+APIs. The Google Play Store has extensive documentation on how to do this, and
+we've also included a high level guide below.
+
+* [Google Play Billing Overview](https://developer.android.com/google/play/billing/billing_overview)
+
+### Android
+
+1. Create a new app in the [Play Developer
+   Console](https://play.google.com/apps/publish/) (PDC).
+
+2. Sign up for a merchant's account in the PDC.
+
+3. Create IAPs in the PDC available for purchase in the app. The example assumes
+   the following SKU IDs exist:
+
+   - `consumable`: A managed product.
+   - `upgrade`: A managed product.
+   - `subscription_silver`: A lower level subscription.
+   - `subscription_gold`: A higher level subscription.
+
+   Make sure that all of the products are set to `ACTIVE`.
+
+4. Update `APP_ID` in `example/android/app/build.gradle` to match your package
+   ID in the PDC.
+
+5. Create an `example/android/keystore.properties` file with all your signing
+   information. `keystore.example.properties` exists as an example to follow.
+   It's impossible to use any of the `BillingClient` APIs from an unsigned APK.
+   See
+   [here](https://developer.android.com/studio/publish/app-signing#secure-shared-keystore)
+   and [here](https://developer.android.com/studio/publish/app-signing#sign-apk)
+   for more information.
```

Comment:
Flutter also has docs on signing apks here https://docs.flutter.dev/deployment/android#signing-the-app